src/App.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+import Courses from '../src/components/Courses/Courses';
+import Header from './components/Header/Header';
+import { mockCourses, mockAuthors } from './data/data';
+import CreateCourse from './components/CreateCourse/CreateCourse';
+
+const App = () => {
+  const [course, setCourse] = React.useState();
```

Comment:
1. For collections/arrays, please, use plural form
2. For collections/arrays it's a good practice to initialise empty collection (`[]` in out case :))